controllers/endpoint/endpoints_controller.go

```diff
@@ -26,7 +26,7 @@ const controllerName = "endpoint"
 
 // Reconciler EndpointReconciler reconciles an Endpoints object
 type Reconciler struct {
-	client.Client
+	Client                  client.Client
```

Comment:
What's the benefit here for explicit declaration as the change is not reflecting elsewhere, is it just for readability?

Reply:
It is a good practice to define an explicit field for `Client`. If you do not do it then 2 things happen:
* golanglint issues warnings
* Semantically methods defined on the Client are for the Client and not on the reconciler. But if you embed `Client` then you can then call things like `reconciler.Get` that semantically does not make much sense. therefore you make it explicit.